scripts/install_xgboost.sh

```diff
@@ -0,0 +1,38 @@
+# Xgboost
+
+set -e
+set -x
+
+apt update && apt install -y software-properties-common
+
+# apt-get update
+# apt-get install -y --allow-downgrades --allow-remove-essential --allow-change-held-packages --allow-unauthenticated --no-install-recommends --no-upgrade \
+#   gcc-8 \
+#   g++-8 \
+#   libnccl2 \
+#   libnccl-dev
+
+# install nvidia toolkit
+apt-key adv --fetch-keys https://developer.download.nvidia.com/compute/cuda/repos/ubuntu2004/x86_64/7fa2af80.pub
+add-apt-repository "deb https://developer.download.nvidia.com/compute/cuda/repos/ubuntu2004/x86_64/ /"
+apt update && apt install -y nvidia-cuda-dev nvidia-cuda-toolkit
+
+# install cmake
+apt update && apt install -y build-essential libssl-dev gcc-8 g++-8
```

Comment:
The other script deletes the cache after running apt update. Is this not necessary?

Reply:
You're right, that second `apt update` in not needed.  I'll remove it in my next commit.  Thanks @eitsupi!